docs/setup/upgrade/resolving-migration-failures.asciidoc

```diff
@@ -133,7 +133,7 @@ Migrations will fail if saved objects belong to an unknown
 saved object type. Unknown saved objects are typically caused by performing manual modifications
 to the {es} index (no longer allowed in 8.x), or by disabling a plugin that had previously created a saved object.
 
-We recommend using the {kibana-ref-all}/7.17/upgrade-assistant.html[Upgrade Assistant]
+We recommend using the {kibana-ref-all}/{version}/upgrade-assistant.html[Upgrade Assistant]
```

Comment:
> We recommend using the {kibana-ref-all}/{version}/upgrade-assistant.html[Upgrade Assistant]

I think this will fail since the "upgrade-assistance" section has been removed in this PR.

Reply:
It seems we actually removed the [documentation](https://github.com/elastic/kibana/blob/7.17/docs/management/upgrade-assistant/index.asciidoc) and need to revive it.

Reply:
I think we can just use the content we already have in the [plugin README](https://github.com/elastic/kibana/blob/main/x-pack/plugins/upgrade_assistant/README.md)

Reply:
For the record, the Upgrade Assistant was re-enabled for all versions as of 8.6

Reply:
> I think we can just use the content we already have in the [plugin README](https://github.com/elastic/kibana/blob/main/x-pack/plugins/upgrade_assistant/README.md?rgh-link-date=2022-12-15T08%3A34%3A53Z)

The README contains a lot of details that, IMO, could be overwhelming for a high-level intro to the UA. I was thinking we could tweak the docs we have in 7.17 and link out to the UA API's from there.